course/constants.py

```diff
@@ -368,6 +368,8 @@ class flow_permission:  # noqa
     set_roll_over_expiration_mode = "set_roll_over_expiration_mode"
     see_session_time = "see_session_time"
     lock_down_as_exam_session = "lock_down_as_exam_session"
+    send_interact_email_to_course_staff =\
+        "send_interact_email_to_course_staff"
```

Comment:
Is that ok? I wish you can polish the naming of the code in order not to cause misunderstanding. Sorry for my bad English.